src/bank/Dime.java

```diff
@@ -0,0 +1,15 @@
+package bank;
+
+public class Dime extends Money {
+
+    public Dime() {
+        super.setvalue(.1);
+        System.out.println("Adding $0.10");
+    }
+
+    public Dime(int num) {
+        double total = num * 0.1;
+        super.setvalue(.1 * num);
+        System.out.println("Adding $" + total);
```

Comment:
this is fine, just looks like the readme wants you to output the quantity of the coin instead
```suggestion
        System.out.println(num + "Dimes");
```